### ISSUE TITLE
MediaType field is not supported on term meta.

### ISSUE THREAD

**Issue by jminguely** [themosis/framework#726]
Hello,

I'm trying to attach a media field to a term from a specific Taxonomy.

```php
TaxonomyField::make($type)
            ->add(Field::media('image'))
            ->set();
```

The code is working perfectly with a `text` field but is returning the following error with a `media` type of field:

```
Field Themosis\Forms\Fields\Types\MediaType is not supported on term meta.
```

**Comment by jlambe** [comment]
Yup the `media` field is not yet compatible for use inside a taxonomy term as well as from an option page, ... it only exists for metabox as a ReactJS component.

You need to create a custom field for use inside a term.

Currently there is a thinking to have in order to provide JS based fields for all screens (from Gutenberg editor, metabox, terms, ...) I still haven't found the right solution so if you have any suggestion, it is welcomed :)
